### DISCUSSION TITLE
How to Get GitHub Education Benefits

### DISCUSSION THREAD

**Discussion by iamimran** [community/community#102115] [GitHub Education] [answered]
### Hi everyone,

Hello,
My name is Imran and I am Lecturer of Computer Science in a Government College in Islamabad, Pakistan. I've applied multiple times for GitHub Education benefits but getting rejected.
Last rejection email suggested that:
1. The text in the photo you used was unable to be read:
    Official ID card which I uploaded is very clear and everything including my photo, name of college and other details are clearly visible and understandable.
2. The image you selected does not appear to contain your school name:
    School/College is also appearing on the official ID card image. (For reference I also uploaded the same image here)
3. Your document does not appear to indicate you as a faculty member:
    My official ID card also clearly mentioned my position (lecturer) and my scale which is BPS-18.
4. Please correct your GitHub billing information:
    Billing information also displays my name as it mentioned on my official ID card.

As I see on my end everything is as per GitHub requirements. Could you please help me what am I missing here. Thanks.
My official ID card is also attached

**Comment by faridnec** [comment, marked as answer]
hey, have you tried direct camera? this issue was solved when I did change my uploaded document by taking direct photos with the camera. They seems sometime does not accept uploaded document

And I would suggest not to post personal information here (like your card) without censor in case if something happens. I hope this helps

**Reply by iamimran** [reply]
Thanks for the response. I haven't tried it yet. But I'll now.

**Reply by Remzi87** [reply]
i don not any with me now

**Reply by Remzi87** [reply]
I don not have any iD  WITH ME NOW

**Comment by minhnnse185018** [comment]
the same problems with the direct camera
